init/eessi_environment_variables

```diff
@@ -15,6 +15,20 @@ function show_msg {
   fi
 }
 
+function check_rhel_ca {
+
+  os=$(grep -oP '(?<=^ID_LIKE=).+' /etc/os-release | tr -d '"')
+
+  version_id=$(grep -oP '(?<=^VERSION_ID=).+' /etc/os-release | tr -d '"')
+  version_major=${version_id%\.*}
+
+  if [[ $os =~ "rhel" ]] && [ $version_major -gt 7 ]
+  then
+        export CURL_CA_BUNDLE=/etc/pki/tls/certs/ca-bundle.crt
```

Comment:
Given that you know the path where the certs should be, isn't it enough to check if it exists and if so export the variable?

Reply:
No need for an OS check with that approach

Reply:
True! I wanted to make sure that nothing breaks, but you are right, my implementation was over-engineered, now its much more simple